public/paths/transaction/transaction.yml

```diff
@@ -0,0 +1,49 @@
+put:
+  tags:
+    - Transaction
+  summary: Update transaction status
+  description: Update a transaction status
```

Comment:
I would add something about that the transaction must be in the status pending to be updated with this endpoint.